src/components/Player/Pages/Matches/Matches.jsx

```diff
@@ -23,47 +24,47 @@ Matches.propTypes = {
   strings: PropTypes.shape({}),
 };
 
-const getData = (props) => {
-  props.getPlayerMatches(props.playerId, props.location.search);
-};
-
-class RequestLayer extends React.Component {
-  static propTypes = {
-    location: PropTypes.shape({
-      key: PropTypes.string,
-    }),
-    playerId: PropTypes.string,
-    strings: PropTypes.shape({}),
-  }
-
-  componentDidMount() {
-    getData(this.props);
+const getData = (props, gamemode) => {
+  let params = props.location.search;
+  if (gamemode === 'turbo') {
+    params = `${props.location.search}&game_mode=23`;
```

Comment:
you'll probably want to apply the game_mode filter in the actions file, so it can be applied to all of the possible /players calls. You also need significant=0